### DISCUSSION TITLE
Generated Pages show 404 error

### DISCUSSION THREAD

**Discussion by FelixPetriconi** [community/community#34572] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi,

I have setup the generation of our page https://mevislab.github.io/examples/ from this source code https://github.com/MeVisLab/examples.

The page is generated and deployed successfully as stated here: https://github.com/MeVisLab/examples/actions/runs/3150407276

The generated result is pushed into the gh-pages branch and this has a index.html file.

But I get still a 404 error on the page URL. 

I have added a README.md and a .nojekyll file as it was mentioned on SO, but I still have no success.

Every help is much appreciated!

Many thanks,
Felix

**Comment by jfullstackdev** [comment, marked as answer]
I just noticed that you don't have the root as well, although this is not required, try enabling that and see, or another sample repo to be enabled by GitHub Pages manually,

![image](https://user-images.githubusercontent.com/47092464/193055843-c96ae79e-7c41-4b1c-bb90-bffd45a20e6b.png)

or maybe this, there is a new Update, try checking that at the repo's setting

![image](https://user-images.githubusercontent.com/47092464/193056364-54fdf345-5a1d-457c-b992-b4bc6f4ec594.png)

**Reply by FelixPetriconi** [reply]
Many thanks!
The problem was again not in the computer but before it. I configured main as the deployment branch and not gh-pages.
